fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -186,6 +186,7 @@ statement
     | showPluginsStatement
     | showRepositoriesStatement
     | showOpenTableStatement
+    | showPrivilegesStatement
```

Comment:
better to arrage it by feature, i.e. put it with other authz statements, like "show grants".

Reply:
I’m sorting them in alphabetical order, why don’t you wait for time to sort them by feature later?